File: core/models/enums.py
from enum import Enum

class PaymentType(Enum):
    EFECTIVO = "Efectivo"
    TARJETA = "Tarjeta"
    CREDITO = "Crédito"
    SIN_ESPECIFICAR = "Sin especificar"

class InventoryMovementType(str, Enum):
    """Types of inventory movements."""
    PURCHASE = "PURCHASE"
    SALE = "SALE"
    ADJUSTMENT = "ADJUSTMENT"
    INITIAL = "INITIAL"

class CashDrawerEntryType(str, Enum):
    """Types of cash drawer entries."""
    IN = "IN"
    OUT = "OUT"
    CLOSE = "CLOSE"

class StandardUnitType(str, Enum):
    """Standard unit types for products."""
    UNIDAD = "Unidad"
    KILOGRAMO = "Kilogramo"
    GRAMO = "Gramo"
    LITRO = "Litro"
    MILILITRO = "Mililitro"
    METRO = "Metro"
    CENTIMETRO = "Centímetro"
    CAJA = "Caja"
    PAQUETE = "Paquete"
    DOCENA = "Docena"
    PAR = "Par"
    ROLLO = "Rollo"
    BOTELLA = "Botella"
    LATA = "Lata"
    BOLSA = "Bolsa"

File: csharp/src/Eleventa.Domain/Entities/Enums.cs
namespace Eleventa.Domain.Entities;

/// <summary>
/// Types of payment methods.
/// </summary>
public enum PaymentType
{
    Efectivo,        // Cash
    Tarjeta,         // Card
    Credito,         // Credit
    SinEspecificar   // Unspecified
}

/// <summary>
/// Types of inventory movements.
/// </summary>
public enum InventoryMovementType
{
    Purchase,
    Sale,
    Adjustment,
    Initial
}

/// <summary>
/// Types of cash drawer entries.
/// </summary>
public enum CashDrawerEntryType
{
    In,
    Out,
    Close
}

/// <summary>
/// Standard unit types for products.
/// </summary>
public enum StandardUnitType
{
    Unidad,
    Kilogramo,
    Gramo,
    Litro,
    Mililitro,
    Metro,
    Centimetro,
    Caja,
    Paquete,
    Docena,
    Par,
    Rollo,
    Botella,
    Lata,
    Bolsa
}
